app/assets/javascripts/discourse/app/components/d-button.js

```diff
@@ -90,6 +94,11 @@ export default Component.extend({
     return computedLabel;
   },
 
+  @discourseComputed("ariaExpanded")
+  computedAriaExpanded(ariaExpanded) {
+    return ariaExpanded && ariaExpanded !== "false" ? "true" : "false";
```

Comment:
Is there a reason why `ariaExpanded` attribute can't be strict and accept just `true` and `false`?

Reply:
I'm on the fence on this...

The reason being, technically it's "true" not true that aria-expanded expects. But I'm fine with allowing only the boolean if we are ok with this. I don't want to only allow "true"/"false" for sure.

Reply:
I guess it won't matter with angle bracket components, since we'll be directly setting `aria-expanded` property there:

`<DButton ... aria-expanded="true" />`

For now, we control the API, so I'd go with just booleans: ;ess code, and will be easier to do search-and-replace later. 😉